tests/governance_js.py

```diff
@@ -432,6 +432,41 @@ def test_operator_proposals_and_votes(network, args):
     return network
 
 
+@reqs.description("Test operator provisioner proposals")
+def test_operator_provisioner_proposals_and_votes(network, args):
+
+    node = network.find_random_node()
+
+    # Assign one member as an operator provisioner
+    operator_provisioner = network.consortium.get_member_by_local_id("member0")
+    network.consortium.set_member_data(
+        node,
+        operator_provisioner.service_id,
+        member_data={"is_operator_provisioner": True},
+    )
+
+    # Create a proposal to assign a member as an operator
+    operator = network.consortium.get_member_by_local_id("member1")
+    set_operator, _ = network.consortium.make_proposal(
+        "set_member_data",
+        member_id=operator.service_id,
+        member_data={"is_operator": True},
+    )
+
+    # Check an operator provisioner can provision operators without a majority of votes
+    with node.client(None, "member0") as c:
+        r = c.post("/gov/proposals", set_operator)
+        assert r.status_code == 200, r.body.text()
+        assert r.body.json()["state"] == "Accepted", r.body.json()
+
+    # Reset the member to an operator for other tests
+    network.consortium.set_member_data(
+        node,
+        operator_provisioner.service_id,
+        member_data={"is_operator": True},
+    )
```

Comment:
Can we have a negative tests where the operator provisioner tries to pass something that's not adding (or removing, which is missing) an operator, and confirm that it doesn't pass automatically?

Reply:
Added the negative test and expanded to include coverage of `remove_member` in https://github.com/microsoft/CCF/pull/4343/commits/5223abe0d0d4449667145610fae7762f9daf688b